components/script/dom/readablestream.rs

```diff
@@ -302,6 +303,19 @@ impl ReadableStream {
         }
     }
 
+    /// https://streams.spec.whatwg.org/#readablestream-storederror
+    #[allow(unsafe_code)]
+    pub fn get_stored_error(&self) -> SafeHandleValue{
+        unsafe {
+            // SafeHandleValue::from_raw(self.stored_error.handle())
+
+            // let cx = GlobalScope::get_cx();
+            // rooted!(in(*cx) let mut rval = UndefinedValue());
+            // self.stored_error.to_jsval(*cx, rval.handle_mut());
+            //
```

Comment:
@gterzian how can I return `SafeHandleValue ` from `stored_error `, I tried to do this:
```
            let cx = GlobalScope::get_cx();
            rooted!(in(*cx) let mut rval = UndefinedValue());
            self.stored_error.to_jsval(*cx, rval.handle_mut());
            rval.handle()
```
but I am missing somehting

Reply:
Thank you!